src/core/crypto.c

```diff
@@ -201,7 +201,7 @@ QuicCryptoInitialize(
 Exit:
 
     if (QUIC_FAILED(Status)) {
-        for (uint8_t i = 0; i < QUIC_PACKET_KEY_COUNT; ++i) {
+        for (size_t i = 0; i < QUIC_PACKET_KEY_COUNT; ++i) {
```

Comment:
Why?

Reply:
My guess is since its indexing, it prefers the size that can index into an array. The generated assembly is the same, as the variable is stuck in a register anyway, so I think this is ok. Less chance for accidental rollover bugs in the future, even though I know QUIC_PACKET_KEY_COUNT likely will not change.

Reply:
Because `QUIC_PACKET_KEY_COUNT` is an `enum`, i.e., an `int`. Could also use `int`.